### ISSUE TITLE
Autogenerate tests for protocol methods from x-ms-examples in swagger

### ISSUE THREAD

**Issue by srnagar** [Azure/autorest.java#1048]
We currently write manual tests for all libraries (including LLC). We should consider generating tests for protocol methods based on the x-ms-examples provided in the swagger.

**Comment by weidongxu-microsoft** [comment]
Yeah, this is a long term goal for codegen. For high-level we would want to generate sample/test code from x-ms-examples (probably start working on it in next month). LLC would likely be easier as it can use the JSON payload directly (instead of generating code with correct type and grammar).

**Comment by weidongxu-microsoft** [comment]
Fluent done in https://github.com/Azure/autorest.java/issues/1077

However there is very little example in data-plane swagger. So probably low priority.
